src/renderers/testing/ReactTestMount.js

```diff
@@ -20,8 +20,8 @@ var getHostComponentFromComposite = require('getHostComponentFromComposite');
 var instantiateReactComponent = require('instantiateReactComponent');
 var invariant = require('invariant');
 
-type TestRendererOptions = {
-  createNodeMock: (element: ReactElement) => Object,
+export type TestRendererOptions = {
+  createNodeMock: (element: ReactElement<any>) => Object,
```

Comment:
Can you put and remove `<any>`

```js
import { ReactElement } from 'ReactElementType';
```

We're trying to use the internal type (and not the one provided by flow) inside of the codebase